core/domain/question_services.py

```diff
@@ -337,7 +339,7 @@ def get_question_from_model(question_model):
     # Migrate the question if it is not using the latest schema version.
     if (question_model.question_state_data_schema_version !=
             feconf.CURRENT_STATE_SCHEMA_VERSION):
-        _migrate_state_schema(versioned_question_state)
+        _migrate_state_schema(versioned_question_state, question_model)
```

Comment:
Maybe pass the linked_skill_ids list fom here itself, instead of passing around the entire model.

Reply:
Done.

Reply:
Not sure if I am seeing an old commit but you are still passing the model right?

Reply:
I'm still passing in question_model here but I changed it to linked_skill_ids in `update_state_from_model` function in question_domain, like what I did for skills.